### DISCUSSION TITLE
🆕  Discussions now out of beta!

### DISCUSSION THREAD

**Discussion by evi-liu** [community/community#5255] [Discussions]
<img src="https://user-images.githubusercontent.com/8809849/130519602-dbcb9ebe-53db-4bd7-b722-55aa146ee8cb.png" width="600"/>

In case you missed it, Discussions is now officially out of public beta! You can read more about the announcement [here](https://github.blog/2021-08-17-github-discussions-out-of-beta/).

Happy Discussion-ing!

**Comment by naser7717** [comment]
N

**Reply by Clithapping2** [reply]
Great

**Comment by TechStudent10** [comment]
Yay!

**Reply by p506** [reply]
Bantu kami
